app/controllers/statements_controller.rb

```diff
@@ -3,7 +3,6 @@ class StatementsController < ApplicationController
   before_filter :authenticate_user!
   before_filter :check_acting_as
   before_filter :init_account
-  before_filter :init_statement, :except => :index
```

Comment:
But yes the `statements#show` route seems obsolete given we have `facility_accounts#show_statement`.